### PR TITLE
Fixed path split in windows

### PR DESCRIPTION
**Pull request by smokingkapor**

### CODE REVIEW

File: gm.js [hunk 1]
```diff
@@ -9,9 +9,13 @@ gm = function() {
...
 
-// Split the path by : or ;
-// XXX: windows is not tested
-var binaryPaths = process.env['PATH'].split(/:|;/);
+// Split the path by : or ; for linux
+// Split the path by ; for windows
+if (/^win/.test(process.platform)) {
+  var binaryPaths = process.env['PATH'].split(/;/);
+} else {
+  var binaryPaths = process.env['PATH'].split(/:|;/);
```
Comment (raix) [inline gm.js]: is `;` still needed on non windows?

Comment (smokingkapor) [inline gm.js]: I'm not sure about the path formation for linux. So I just keep it.

**Comment by aldeed** [comment]
Fine with me to merge

**Comment by smokingkapor** [comment]
I just checked the path formation for linux, ':' is enough for linux. I will update it.

**Comment by smokingkapor** [comment]
Done. Can u help to merge and release a new version. Thanks.

**Comment by raix** [comment]
Thanks @smokingkapor, getting ready for windows :)

**Pull request merged**